LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Blocknative Corp
+Copyright (c) 2020 Matter Labs Corp
```

Comment:
I don't think we can easily get rid of the copyright. The right statement will be `Copyright (c) 2020 Matter Labs, Blocknative Corp`. Also, Matter Labs is not a corp anyways.